common/views/components/TransitionLink/index.tsx

```diff
@@ -0,0 +1,33 @@
+import Link from 'next/link';
+import { useRouter } from 'next/router';
+import { SyntheticEvent } from 'react';
+
+declare global {
+  interface Document {
+    startViewTransition: (cb: () => void) => void;
+  }
+}
+
+const TransitionLink = ({ href, children, ...props }) => {
```

Comment:
Would it be useful to type some of this? Thinking `href` which is currently `any`

Reply:
and `link` in `content/webapp/components/GuideStopCard/index.tsx` is `string | undefined`, so it would catch those cases?